openstates/de/committees.py

```diff
@@ -22,29 +22,41 @@ def scrape(self, chamber, term):
 
         url = urls[chamber] % (session,)
```

Comment:
What about the [Joint Finance Committee](http://www.legis.delaware.gov/legislature.nsf/Lookup/CGO_JFCDescription), which I think is the only joint committee in the state legislature? It's partially scraped as a [House "committee,"](http://legis.delaware.gov/LIS/LIS148.NSF/*/D4ABF3766FDC0A8485257DC400542E52/?opendocument&nav=House) but it'd be nice to make this a full scrape as `joint` and throw out the partial House scrape.